po/glossary/el.po

```diff
@@ -45,117 +46,104 @@ msgid "account name"
 msgstr "όνομα λογαριασμού"
 
 #. "The left side of the balance sheet in T account form shows the application of funds in form of assets. Because it contains only assets use assets directly. Complement: Passive. See also: Report Form"
-#, fuzzy
 msgid "account type: Active"
-msgstr "Είδος Παρασκηνίου:"
+msgstr "Είδος λογαριασμού: Ενεργός"
```

Comment:
@giorgos georgopoulakis: Translate only the part behind the colon! Later, too.

Reply:
you pinged the wrong giorgos here :)